i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,47 @@ sidebar_position: 4
-- *Code business logic `store` / `model` / `state` / ...*
+## Naming in Feature-Sliced Design {#naming-in-fsd}
 
-## Naming in FSD
+The methodology uses specific terms such as:
 
-The methodology uses such terms as
+- "app", "process", "page", "feature", "entity", "shared" as layer names,
+- "ui', "model", "lib", "api", "config" as segment names.
 
-- `app`, `process`, `page`, `feature`, `entity`, `shared` - *[layers][refs-layers]*
-- `ui`, `model`, `lib`, `api` - *[segments][refs-segments]*
+Original naming is crucial to prevent confusion among team members and new developers joining the project. Using unique names also helps when seeking help from the community.
 
-Within the framework of the methodology, each of these terms has a [clear definition][refs-reference]
+## Naming Conflicts {#when-can-naming-interfere}
 
-When developing a project using the **Feature-Sliced Design** methodology, it is very important [to adhere to the original naming, in order to avoid misunderstandings][disc-src] both among the team members and outside of it.
+Naming conflicts can occur when terms used in the FSD methodology overlap with terms used in the business:
 
-- If a new developer comes to the project who is familiar with the methodology, **he should see the terms already familiar to him**
-- If you ask for help in the community, you will get an answer to your question faster, **if you use the same terminology**
+- `FSD#process` vs simulated process in an application,
+- `FSD#page` vs log page,
+- `FSD#model` vs car model.
 
-## When can naming interfere?
+For example, a developer seeing the word process in the code will spend unnecessary time trying to understand which process they are talking about.
 
-When developing a project for displaying/building/modeling any processes, or developing an application for the layout of magazine pages, you may face the problem that **the terms used in the methodology overlap with the terms that your business operates**.
+These **collisions can disrupt the development process**, as developers may spend extra time trying to figure out exactly what they are talking about.
 
-- `FSD#process` vs simulated process in your application
-- `FSD#page` vs magazine page
-- `FSD#model` vs car model
+When the project glossary contains terminology specific to FSD, it is critical to exercise caution when discussing these terms with the team and non-technical stakeholders.
 
-<!-- TODO: think about examples for other terms -->
+To communicate effectively with the team, it is recommended that the abbreviation "FSD" be used to prefix the methodology terms. For example, when talking about a process, you might say, "We can put this process at the FSD level."
 
-Such name collisions can negatively affect the development process.
+Conversely, when communicating with non-technical stakeholders, it is best to limit the use of FSD terminology and refrain from mentioning the internal structure.
```

Comment:
issue (non-blocking): I think it's a bit unclear here what internal structure we're talking about

suggestion:
```suggestion
Conversely, when communicating with non-technical stakeholders, it is best to limit the use of FSD terminology and refrain from mentioning the internal structure of the codebase.
```

Reply:
Fixed in latest commit.